libbeat/reader/readfile/strip_newline.go

```diff
@@ -42,6 +42,11 @@ func (p *StripNewline) Next() (reader.Message, error) {
 	L := message.Content
 	message.Content = L[:len(L)-lineEndingChars(L)]
 
+	if len(message.Original) > 0 {
+		O := message.Original
+		message.Original = O[:len(O)-lineEndingChars(O)]
```

Comment:
Minor detail, but maybe we don't want to strip the newline here? Right now the multiline reader inserts a separator, after this one has been removed. => we don't report the original, but some modified contents. We can simply concatenate `message.Original` if we were not strip newlines here.